### PR TITLE
Extended the frontend to match the design for pick a track

### PR DESCRIPTION
**Pull request by ankita-sri**
(dates added to pick your track - in bold)

### CODE REVIEW

File: frontend/src/components/shared/pick-track/index.tsx [hunk 1]
```diff
@@ -122,4 +126,4 @@ class PickTrack extends React.Component<IPickTrackProps, IPickTrackState> {
   }
 }
 
-export default PickTrack;
+export default PickTrack
```
Comment (lustoykov) [inline frontend/src/components/shared/pick-track/index.tsx]: @ankita-sri can you re-add the semicolon here? (and amend the commit)

Comment (ankita-sri) [reply frontend/src/components/shared/pick-track/index.tsx]: did 🗡

Comment (lustoykov) [reply frontend/src/components/shared/pick-track/index.tsx]: @ankita-sri squash commits please :P

Comment (lustoykov) [reply frontend/src/components/shared/pick-track/index.tsx]: ping @ankita-sri

**Pull request merged**